app/controllers/api/v2/users_controller.rb

```diff
@@ -12,8 +12,8 @@ class UsersController < V2::BaseController
         ['compute_attributes']
 
       before_action :find_optional_nested_object
-      skip_before_action :authorize, :only => [:extlogin]
-      before_action :authenticate, :only => [:extlogin]
+      skip_before_action :authorize, :only => [:extlogin, :invalidate_jwt]
```

Comment:
Why do you need to skip authorization for `invalidate_jwt` action?
You can specify the action in the security block: https://github.com/theforeman/foreman/blob/eb66c09e63ba9b04f3d86c240ec25de52647bc84/config/initializers/f_foreman_permissions.rb#L566
This will make sure the current user has the `edit_users` permission before accessing the endpoint.